.github/workflows/containers.yaml

```diff
@@ -35,9 +35,14 @@ jobs:
         run: |
           source ci/config/env
           echo ::set-output name=PARENT_IMG::${PARENT_IMG}
+          IPA_VERSION="$( docker run --rm --entrypoint "" "${PARENT_IMG}" rpm -q --qf '%{version}' freeipa-server )"
```

Comment:
Try to retrieve this information from image metadata if it were available.

Reply:
The PARENT_IMG does not have metadata information about the IPA version, so I leave it as it is.